### PR TITLE
[16.0][MIG] stock_move_line_auto_fill

### PR DESCRIPTION
**Pull request by syera94**
@florian-dacosta @bealdav hello guys, could you check this PR realized thanks to Florian :)

### CODE REVIEW

**Review by florian-dacosta — APPROVED** [review]
LGTM code review

File: stock_move_line_auto_fill/views/stock_picking.xml [hunk 1]
```diff
@@ -16,9 +15,18 @@
...
+                />
+                <field
+                    name="action_pack_op_auto_fill_allowed"
+                    invisible="1"
+                    groups="stock.group_stock_user"
+                />
+                <field
+                    name="auto_fill_operation"
+                    invisible="1"
+                    groups="stock.group_stock_user"
```
Comment (bealdav) [inline stock_move_line_auto_fill/views/stock_picking.xml]: Not sure here and above but if field is invisible, define a specific group have no effect !?

**Review by flotho — APPROVED** [review]
Tested on runboat,
LGTM
Thanks for this migration

File: stock_move_line_auto_fill/views/stock_picking.xml (hunk 1 above)
Comment (florian-dacosta) [reply stock_move_line_auto_fill/views/stock_picking.xml]: indeed, must be useless
@syera94 Can you remove these groups on the invisible fields ?

**Review by bealdav — APPROVED** [review]

**Pull request merged**